buildconfig/stubs/pygame/_sdl2/__init__.pyi

```diff
@@ -0,0 +1,3 @@
+from pygame._sdl2.audio import *
```

Comment:
This should probably contain all the modules _sdl2 has, not just these. The missing ones are `touch`, `mixer` and `controller` (real `__init__.py` is in `src_py/_sdl2`).

Reply:
Good catch

Reply:
I think I'll skip this change in this PR. Because fixing this in the stubs also involves fixing `src_py/_sdl2/__init__.py` to add all those imports. I want to keep code changes to the minimal in this PR. While this is just a three line thing, it's an API decision to make, and that is better made by someone else more familiar with `_sdl2` in a future PR, and not myself :)